packages/dynamodb/index.test-d.ts

```diff
@@ -21,21 +17,54 @@ const options = {
   awsClientAssumeRole: 'some-role',
   awsClientCapture: captureAWSv3Client,
   fetchData: {
-    config: {
-      Application: 'app',
-      ClientId: '0001',
-      Configuration: 'lambda-n',
-      Environment: 'development'
+    superSecretAccessToken: {
```

Comment:
secrets should never be stored in dynamo, should use an example where a config for the lambda is passed in instead.

Reply:
Good point 😎